modules/ELT/pages/getting-started-with-airbyte.adoc

```diff
@@ -0,0 +1,197 @@
+= Airbyte with Teradata Vantage
+:experimental:
+:page-author: Krutik Pathak
+:page-email: krutik.pathak@teradata.com
+:page-revdate: June 1st, 2023
+:description: Use Airbyte with Teradata Vantage.
+:keywords: airbyte, data warehouses, compute storage separation, teradata, vantage, cloud data platform, object storage, data transfer, data extraction, enterprise analytics, elt.
+:tabs:
+:dir: getting-started-with-airbyte
+
+== Overview
+
+This tutorial showcases how to use Airbyte (an Open-source Extract Load Transform Tool) with Teradata Vantage. We work with a very simple end-to-end setup to load the employees’ data from Google Sheets to Teradata Vantage using Airbyte.
+
+
+image::{dir}/sample_employees_payrate_google_sheets.png[Sample Employees Payrate Google Sheets, width=75%]
+
+* Source: Google Sheets
+* Destination: Teradata Vantage
+
+== Prerequsites
+* Access to Teradata Vantage Instance. As this will be our destination. You will need a database host, username, and password for Airbyte’s configuration.
```

Comment:
```suggestion
* Access to Teradata Vantage Instance, as this will be the destination. You will need a database host, username, and password for Airbyte’s configuration.
```